Content update: Adding x402 to Payments content (#17007)
ethereum/ethereum-org-website: public/content/payments/index.md

> "Non-custodial" refers to wallets where funds are managed without third-party custody, often through smart contracts.
> While all self-custodial cards are non-custodial, not all non-custodial cards are self-custodial.

> ## Micro-payments for websites (x402) {#x402}

Reviewer: ```suggestion
## Micro-payments for websites & agents (x402) {#x402}
```

> [x402](https://www.x402.org/) is a payment protocol built on Ethereum. It allows for simple, per-use payment using [stablecoins](/stablecoins). This standard makes it economical to pay directly for a single action, such as reading a news article or calling an API, rather than managing subscriptions or “paying” by giving attention to advertising

Reviewer: ```suggestion
[x402](https://www.x402.org/) is an open payment standard that brings native per-use payments to the web. By using [stablecoins](/stablecoins) on low-cost [Ethereum Layer 2 networks](/layer-2), the x402 standard makes it economical for humans and machines to pay directly for a single action, such as reading a news article or calling an API, rather than managing API keys, subscriptions, or “paying” by giving attention to advertising.
* **Removing paywalls and logins:** Instead of creating an account and sharing personal information to read one news article, your wallet can pay the few cents required to unlock it. 
* **Payments for AI agents:** x402 enables autonomous software ("AI Agents") to pay for the data and API calls they need to function, without human intervention. 
```

> 

Reviewer: ```suggestion

### How the x402 payment standard works
```

[x402](https://www.x402.org/) is an open payment standard that brings native per-use payments to the web. By using [stablecoins](/stablecoins/) on low-cost [Ethereum layer 2 networks](/layer-2/), the x402 standard makes it economical for humans and machines to pay directly for a single action, such as reading a news article or calling an API, rather than managing API keys, subscriptions, or “paying” by giving attention to advertising.

- **Removing paywalls and logins:** Instead of creating an account and sharing personal information to read one news article, your wallet can pay the few cents required to unlock it.
- **Payments for AI agents:** x402 enables autonomous software ("AI Agents") to pay for the data and API calls they need to function, without human intervention.

### How the x402 payment standard works {#x402-how}

When a client requests a resource, the server sends a `402 Payment Required` error code along with payment instructions (price, account, and what tokens and chains are supported).

- Your [wallet](/wallets/) detects the request and handles the payment (often with a single click to approve, or automatically using a pre-approved allowance)

> - AI agents with access to pre-approved wallet balances can automatically detect the price and pay instantly to access data or services

Reviewer 2: Could consider a link to our AI Agents page here but also want to be cautious of overwhelming with links:

```suggestion
- [AI agents](/ai-agents/) with access to pre-approved wallet balances can automatically detect the price and pay instantly to access data or services
```
